rpmconf/Install/Util/utilfunc.sh

```diff
@@ -2317,17 +2317,35 @@ getInstallPackages() {
 
     # askInstallPkgYN args : PROMPT REQUIRE_STORE=yes|no YES_STORE_DEFAULT=Y|N NO_STORE_DEFAULT=Y|N
 
+    ZIMBRAINTERNAL=no
+    echo $HOSTNAME | egrep -qe 'eng.zimbra.com$|lab.zimbra.com$' > /dev/null 2>&1
```

Comment:
we should also consider zimbradev.com to be internal vm

Reply:
yes, will add it.

Reply:
in https://github.com/Zimbra/zm-build/blob/aa22561d5e70f835c81a24443fe49b631b62d6e6/rpmconf/Install/Util/globals.sh#L84 we have added eng.synacor.com also not sure why

Reply:
I am also not sure , this is added a long time back.